app/jobs/import_versions_job.rb

```diff
@@ -38,6 +43,7 @@ def import_raw_data(raw_data)
                                      else
                                        "Row #{row}: Unknown error occurred"
                                      end
+        Rails.logger.error "Import {@import} Row #{row}: #{error.message}"
```

Comment:
Missing `#` before `{@import}`. Also, it should be `#{@import.id}`, otherwise you’ll get the actual underlying Ruby object identifier, e.g. `#<Import:0x007f966df84ea8>`.

Reply:
Fixed, force-pushed.